ngraph/frontend/onnx/frontend/src/op/gather.hpp

```diff
@@ -6,8 +6,8 @@
 
 #include <memory>
 
-#include "default_opset.hpp"
 #include "ngraph/node.hpp"
+#include "ngraph/opsets/opset8.hpp"
```

Comment:
Why no more default?

Reply:
Current default points to opset7. I'll file a ticket to upgrade default opset to 8.